jpa/eclipselink.jpa.testapps/jpa.test.batchfetch/src/main/java/org/eclipse/persistence/jpa/testapps/batchfetch/BatchFetchTableCreator.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+// Contributors:
+
+package org.eclipse.persistence.jpa.testapps.batchfetch;
+
+import org.eclipse.persistence.tools.schemaframework.FieldDefinition;
+import org.eclipse.persistence.tools.schemaframework.TableCreator;
+import org.eclipse.persistence.tools.schemaframework.TableDefinition;
+
+public class BatchFetchTableCreator extends TableCreator {
+    public BatchFetchTableCreator() {
+        setName("BatchFetchProject");
+
+        addTableDefinition(buildCompanyTable());
+        addTableDefinition(buildEmployeeTable());
+        addTableDefinition(buildRecordTable());
+    }
+
+    public TableDefinition buildRecordTable() {
+        TableDefinition table = new TableDefinition();
+        table.setName("RECORD");
```

Comment:
I'd like avoid this too generic name to avoid collision with other tests. Please apply there prefix `BATCH_IN_` -> `BATCH_IN_RECORD`.
This comment can be applied to all new DB tables `COMPANY`, `EMPLOYEE`.

Reply:
Prefix added